UI Session/UI Session4/index.html

```diff
@@ -0,0 +1,247 @@
+                <span class="text-cloudy">Cloudy</span>
+                <div class="screenshotDescription">
+                    <img src="./images/cloud1.png" alt="Cloud">
+                    <h2 class="screenshotHeading">When the Clouds</h2>
+                    <p class="aboutScreenshot">Variable information on the air humidity,
+                        the feeling of the weather, and the ability to share this with your friends.
+                    </p>
+                </div>
+            </div>
+
+            <div class="middleScreenshot">
+                <div class="screenshotDescription">
+                    <img src="./images/sun1.png" alt="Sun">
+                    <h2 class="screenshotHeading">When the Sunny</h2>
+                    <p class="aboutScreenshot">Variable information on the air humidity,
+                        the feeling of the weather, and the ability to share this with your friends.
+                    </p>
+                </div>
+                <span class="text-sunny">Sunny</span>
+                <img src="./images/screen02.png" alt="Sunny" class="phone">
+            </div>
+
+            <div class="bottomScreenshot">
+                
+                <img src="./images/screen03.png" alt="Rainy" class="phone">
+                <span class="text-rainy">Rainy</span>
+                <div class="screenshotDescription">
+                    <img src="./images/rainy1.png" alt="Rain">
+                    <h2 class="screenshotHeading">When the Rain</h2>
+                    <p class="aboutScreenshot">Variable information on the air humidity,
+                        the feeling of the weather, and the ability to share this with your friends.
+                    </p>
+                </div>
+            </div>
+        </section>
+
+        <!-- Reviews section -->
+        <section class="reviewSection" id="reviews">
+            <header>
+                <h2>What reviews are saying</h2>
+                <p>Reviews</p>
+            </header>
+
+            <div class="row1Reviews">
+                <blockquote>
+                    <i class="fas fa-quote-left"></i>
+                    <p class="review">
+                        What art offers is space - a certain breathing
+                        room for the spirit.
+                    </p>
+                    <p class="reviewerName">John Updike</p>
+                    <i class="fas fa-quote-right"></i>
+                </blockquote>
+
+                <blockquote>
+                    <i class="fas fa-quote-left"></i>
+                    <p class="review">
+                        What art offers is space - a certain breathing
+                        room for the spirit.
+                    </p>
+                    <p class="reviewerName">John Updike</p>
+                    <i class="fas fa-quote-right"></i>
+                </blockquote>
+            </div>
+
+            <div class="row2Reviews">
```

Comment:
Please search on internet if we can give class name like row2Reviews ? I think it should be row-to-review something like that.